### PR TITLE
chore: bump primitive types

### PR DESCRIPTION
**Pull request by gakonst**
as title - blocked on https://github.com/rust-blockchain/ethereum/issues/39

### CODE REVIEW

**Review by CryptoGodfatherVA2 — APPROVED** [review]
var5

**Review by CryptoGodfatherVA2 — APPROVED** [review]
var5

**Comment by KaoImin** [comment]
When will this PR be merged, the blocking issue has already been done.

File: Cargo.toml [hunk 1]
```diff
@@ -15,12 +15,12 @@ evm-gasometer = { version = "0.33", path = "gasometer", default-features = false
 evm-runtime = { version = "0.33", path = "runtime", default-features = false }
 sha3 = { version = "0.8", default-features = false }
 rlp = { version = "0.5", default-features = false }
-primitive-types = { version = "0.10", default-features = false, features = ["rlp"] }
+primitive-types = { version = "0.11.1", default-features = false, features = ["rlp"] }
 serde = { version = "1.0", default-features = false, features = ["derive"], optional = true }
 codec = { package = "parity-scale-codec", version = "2.0", default-features = false, features = ["derive"], optional = true }
-ethereum = { version = ">= 0.10, <= 0.11", default-features = false }
+ethereum = { git = "https://github.com/rust-blockchain/ethereum", version = "0.12.0", default-features = false }
```
Comment (sorpaas) [inline Cargo.toml:21]: Please remove the git one.

File: Cargo.toml [hunk 2]
```diff
@@ -32,10 +32,10 @@ name = "loop"
 harness = false
 
 [features]
-default = ["std"]
+default = ["std", "scale-info/std", "codec/std"]
```
Comment (sorpaas) [inline Cargo.toml:35]: Add those to `std` feature, not `default`.

File: core/Cargo.toml [hunk 3]
```diff
@@ -9,18 +9,18 @@ keywords = ["no_std", "ethereum"]
...
+scale-info = { version = "2.0.0", default-features = false, features = ["derive"], optional = true }
 # Work-around for https://github.com/myrrlyn/funty/issues/3
 funty = { version = "=1.1", default-features = false }
 
 [dev-dependencies]
 hex = "0.4"
 
 [features]
-default = ["std"]
+default = ["std", "scale-info/std"]
```
Comment (sorpaas) [inline core/Cargo.toml:23]: Same as above.

**Review by CryptoGodfatherVA2 — APPROVED** [review]

**Pull request closed without merging**